news/update_docker_for_circleci.rst

```diff
@@ -0,0 +1,13 @@
+**Added:**
+
+* Add hdf5-tools and future as dependency for docker images used in CircleCI.
```

Comment:
Could you just briefly explain why you need to add `future` as a python dependency ? (basically what you said on slack...)

Reply:
I updated the reason of adding these two dependencies in news file.